ui/app/components/ui/sender-to-recipient/index.scss

```
@@ -17,6 +17,10 @@
 
       &__tooltip-container {
         max-width: 100%;
+        &--icon {
```

Nit: This doesn't seem very BEM like - `icon` doesn't seem like a modifier.

Maybe this is better?
```suggestion
        &-icon {
```

---

replicated this pretty heavily. i'll resolve it throughout.